shared/AppInsightsCore/Tests/Selenium/ApplicationInsightsCore.Tests.ts

```diff
@@ -593,6 +593,7 @@ export class ApplicationInsightsCoreTests extends AITestClass {
 
         this.testCase({
             name: 'newId check randomness',
+            timeout: 15000,
```

Comment:
Default is 5 seconds, but was seeing some test take longer on the build servers -- so changing to 15 seconds.